debian/xenial/kubeadm/debian/postinst

```diff
@@ -19,6 +19,11 @@ set -o nounset
 
 case "$1" in
     configure)
+        # because kubeadm package adds kubelet drop-ins, we must daemon-reload
+        # and restart kubelet now. restarting kubelet is ok because kubelet
+        # postinst configure step auto-starts it.
+        systemctl daemon-reload
```

Comment:
If we do a daemon-reload without the restart, will it pick up config changes on a crash loop?